flow/embedded_views.h

```diff
@@ -179,11 +179,13 @@ class EmbeddedViewParams {
 // in this case ExternalViewEmbedder is a reference to the
 // FlutterPlatformViewsController which is owned by FlutterViewController.
 class ExternalViewEmbedder {
-  // TODO(cyanglaz): Make embedder own the `EmbeddedViewParams`.
 
  public:
   ExternalViewEmbedder() = default;
 
+  // the params object used to composite the platform views.
+  EmbeddedViewParams embedded_view_params;
```

Comment:
The ExternalViewEmbedder is the objects that's responsible for embedding all platform views, it's a little strange to have it keep the parameters of a specific view here.

Discussed offline: we probably want to std:move the params in platform_view_layer.cc